lib/vdc/DataMgr.cpp

```diff
@@ -547,11 +547,18 @@ bool is_int(std::string str) {
 
 template <class T>
 void _sanitizeFloats(T *buffer, size_t n) {
-	for (size_t i=0; i<n; i++) {
+	for (size_t i = 0; i < n; i++) {
 		if (isnan(buffer[i])) buffer[i] = std::numeric_limits<T>::infinity();
 	}
```

Comment:
Can we try to specify the namespace, i.e., `std::isnan()`, and also include the required header, i.e., `#include <cmath>`, and then check again if it behaves as expected?

Reply:
We are currently including cmath.  Adding the namespace qualifier results in the same error.